src/app/admin/_components/faq/hooks/useActiveFaqs.js

```diff
@@ -0,0 +1,18 @@
+import { useGetList } from 'react-admin';
+
+export const useActiveFaqs = ({ page, perPage } = { page: 1, perPage: 50 }) => {
+  const {
+    data: faqs = [],
+    isLoading,
+    error,
+  } = useGetList('faq', {
+    pagination: { page, perPage },
+  });
+  if (isLoading || error) {
+    return { total: 0, data: [], isLoading, error };
+  }
+
+  const sorted = faqs.map(({ id, isActive }) => ({ id, isActive })).sort(a => a.isActive);
+  const filtered = sorted.filter(({ isActive }) => isActive);
```

Comment:
This sorting looks useless because you sort by isActive and after that filter out not Active.

Can't it be 'filter' and 'map',  or 'map' and 'filter', or better 'reduce' func?

Reply:
this sort was needed in case of logging/debugging, but i can remove it no prob

Reply:
done

Reply:
Great!
But why variable named `sorted` if there is no sorting there 🤔?